docs/spectrum/spectrum.rst

```diff
@@ -158,7 +158,7 @@ Spectrum conditions and get Spectrum that suits specific parameters
 
 .. _label_spectral_quantities:
 
-Spectral quantities
+spectral arrays
 ===================
 
 A :class:`~radis.spectrum.spectrum.Spectrum` object can contain one spectral quantity, such as
```

Comment:
`A :class: ~radis.spectrum.spectrum.Spectrum object can contain one spectral **quantity**`

You missed one?

Reply:
I just realized it's not just on spectral quantity is in many different forms ( for, e.g., spectral_quantity, spectral quantities, etc.)